ehcache-core/src/main/java/org/ehcache/core/config/DefaultConfiguration.java

```diff
@@ -56,6 +60,7 @@ public DefaultConfiguration(Configuration cfg) {
     this.caches = new ConcurrentHashMap<>(cfg.getCacheConfigurations());
     this.services = unmodifiableCollection(cfg.getServiceCreationConfigurations());
     this.classLoader = cfg.getClassLoader();
+    this.sharedResourcePools = new ConcurrentHashMap<>(cfg.getSharedResourcePools());
```

Comment:
I elected to implement the config manager's shared resource pools outside of its service implementation.  If this should be refactored to be included as a new service then please let me know.